src/Spell.java

```diff
@@ -0,0 +1,87 @@
+import java.util.Objects;
+
+public class Spell {
```

Comment:
this has to implement the Ability interface

Reply:
Sorry! I forget that